homeassistant/components/garages_amsterdam/config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Config flow for Garages Amsterdam integration."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from . import get_coordinator
+from .const import DOMAIN  # pylint:disable=unused-import
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Garages Amsterdam."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    _options = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        errors = {}
+
+        if self._options is None:
+            self._options = {}
+            coordinator = await get_coordinator(self.hass)
```

Comment:
We try to avoid storing things in hass.data from the config flow. Is that needed or can we just get the client session and call `garages_amsterdam.get_cases` instead of using the coordinator?

Reply:
We also need to handle connection errors here.